e2e/tests/005.sh

```diff
@@ -27,20 +27,18 @@ source "${LIBDIR}/k8s.sh"
 
 kubeconfig="$HOME/.kube/config"
 
-# apply both AMF and SMF so they both start processing
-k8s_apply "$kubeconfig" "$TESTDIR/005-regional-free5gc-amf.yaml"
-k8s_apply "$kubeconfig" "$TESTDIR/005-regional-free5gc-smf.yaml"
+k8s_apply "$kubeconfig" "$TESTDIR/005-edge-free5gc-upf.yaml"
 
-cluster_kubeconfig=$(k8s_get_capi_kubeconfig "$kubeconfig" "default" "regional")
+for cluster in "edge01" "edge02"; do
+    k8s_wait_exists "$kubeconfig" 600 "default" "packagevariant" "edge-free5gc-upf-${cluster}-free5gc-upf"
+done
 
-# check the AMF
-k8s_wait_exists "$kubeconfig" 600 "default" "packagevariant" "regional-free5gc-amf-regional-free5gc-amf"
-k8s_wait_ready "$kubeconfig" 600 "default" "packagevariant" "regional-free5gc-amf-regional-free5gc-amf"
-k8s_wait_exists "$cluster_kubeconfig" 600 "free5gc-cp" "deployment" "amf-regional"
-k8s_wait_ready_replicas "$cluster_kubeconfig" 600 "free5gc-cp" "deployment" "amf-regional"
+for cluster in "edge01" "edge02"; do
+    k8s_wait_ready "$kubeconfig" 600 "default" "packagevariant" "edge-free5gc-upf-${cluster}-free5gc-upf"
+done
 
-# check the SMF
-k8s_wait_ready "$kubeconfig" 600 "default" "packagevariant" "regional-free5gc-smf-regional-free5gc-smf"
-k8s_wait_exists "$kubeconfig" 600 "default" "packagevariant" "regional-free5gc-smf-regional-free5gc-smf"
-k8s_wait_exists "$cluster_kubeconfig" 600 "free5gc-cp" "deployment" "smf-regional"
-k8s_wait_ready_replicas "$cluster_kubeconfig" 600 "free5gc-cp" "deployment" "smf-regional"
+for cluster in "edge01" "edge02"; do
+    cluster_kubeconfig=$(k8s_get_capi_kubeconfig "$kubeconfig" "default" "$cluster")
+    k8s_wait_exists "$cluster_kubeconfig" 600 "free5gc-upf" "deployment" "free5gc-upf"
```

Comment:
k8s_wait_exists "$cluster_kubeconfig" 600 "free5gc-upf" "deployment" "upf-${cluster}"

Reply:
this fixed the issue for me with the checks

Reply:
yes, I pushed that fix, thanks. So close :)